vcpkg/TOC.yml

```diff
@@ -232,6 +232,8 @@
         href: maintainers/functions/internal/z_vcpkg_setup_pkgconfig_path.md
   - name: CONTROL Files (deprecated)
     href: maintainers/control-files.md
+- name: GitHub Integration

```

Comment:
My 2c, based on your concern about putting this at the top level of the TOC, is that we don't put anything at the top level of the TOC unless it is absolutely clear that it belongs there. If there's any question, it goes somewhere else.

The reason is that the top level of your TOC defines the architecture of your doc set, which in addition to providing critical navigation hints about how things are organized, it also indicates the primary tasks and functions of the thing you are documenting. 
If github integration is a primary, top-level concept/task then it could make sense at the top level. If it isn't, then I'd find concepts it is related to and put it above/below them depending on whether GitHub integration is a superset or subset of those concepts/tasks.

Reply:
I suggest moving this under `Consuming packages`.

Reply:
My inclination is to have an `Integrations` top-level node and to put rationale/motivation documents for each integration partner there. Those would then point to more concrete usage documentation (like what I've added at the top of the new file for the binary caching integration). Ultimately, I'd like the feature flag documentation (which currently doesn't seem to exist) to exist in the "Using" section and we'd just point at that for the practical matters of using the integration point.

Basically, there's lots of missing pieces in the documentation, and so I've run into a situation where I don't want to drastically expand the scope of this, but also want to make it somewhat forward-compatible for when that documentation appears.

Reply:
Moved.